permissions/plugin-linchpin.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "linchpin"
+github: "jenkinsci/linchpin-plugin"
+paths:
+- "io/jenkins/plugins/linchpin-plugin"
```

Comment:
This does not match groupId/pluginId. Please see this repository's readme.

Reply:
Fixed, thanks!